hs_discover/src/components/Search.vue

```diff
@@ -293,9 +293,9 @@ export default {
       countAvailabilities: {},
       availabilityFilter: [],
       resIconName: {
-        'Composite Resource': '/static/static/img/resource-icons/composite48x48.png',
-        'Collection Resource': '/static/static/img/resource-icons/collection48x48.png',
-        'Web App Resource': '/static/static/img/resource-icons/webapp48x48.png',
+        Resource: '/static/static/img/resource-icons/composite48x48.png',
```

Comment:
missing ''

Reply:
Since the key name doesn't have space quotes not needed.

Reply:
right! nice!